server/src/BitBakeProjectScanner.ts

```diff
@@ -91,7 +91,7 @@ export class BitBakeProjectScanner {
   }
 
   rescanProject (): void {
-    logger.info(`request rescanProject ${this._bitbakeDriver.bitbakeSettings.pathToBuildFolder}`)
+    logger.info('request rescanProject')
```

Comment:
This comes from Eugen's version. The command is "rescan" because a first scan is always started at extension activation. I'd rather not focus on this immediately. I have more commits on another branch that would need a lot of rebasing. If you think this naming is really important, please open a ticket for later.